### ISSUE TITLE
Reproduce the result on WMT14 en-de 

### ISSUE THREAD

**Issue by Zrachel** [facebookresearch/fairseq-lua#112]
I have followed the settings on convs2s paper, but cannot reproduce the same results.
While table in the paper shows wmt en-de achieves at bleu 25.16, I get 24.12 at best. 

The difference is I use 8cards while the pretrained model README.md sets `-ngpu=1`. As using one card is so slow, I want to ask is it normal to get 1 point worse for 8cards?

Thank you.

**Comment by michaelauli** [comment]
The instructions in the README are for a single GPU setup, you used eight cards which increases the batch size and requires other settings to be adapted. Please try to increase the learning rate to 0.5, 0.75, 1, or even 1.25. Also, rebase to make sure you use the more efficient batching strategy we pushed recently: https://github.com/facebookresearch/fairseq/commit/d5f8cf69ed3f3c5302ddf1f5585529ad8871e549
Finally, make sure the model trains for a certain number of epochs before annealing by setting minepochtoanneal (say to 50). Also, set -maxbatch to 1920 or larger if you can. 

The PyTorch version of fairseq (https://github.com/facebookresearch/fairseq-py) also has a pre-trained WMT14 en-de model and its' README has instructions for eight gpu training.
